web-modules/javax-servlets-2/src/test/java/com/baeldung/setparam/SetParameterRequestWrapperUnitTest.java

```diff
@@ -55,7 +55,7 @@ public void whenSetValueToWrapperParametersStringArray_thenThe2ndCallShouldNotEq
 
         firstCallValues[0] = NEW_VALUE;
         String[] secondCallValues = wrapper.getParameterValues("input");
-        assertNotEquals(firstCallValues, secondCallValues);
+        assertThat(firstCallValues).isNotEqualTo(secondCallValues);
```

Comment:
I believe @KevinGilmore might have been trying to suggest you to do something like this:
```suggestion
        assertThat(secondCallValues).doesNotContain(NEW_VALUE);
```

Reply:
isNotEqualTo is correct.
We are asserting whether the 2nd call of getParaemterValues() is not equal to the modified value from the 1st call.

Reply:
Assertion updated